python/docs/source/development/debugging.rst

```diff
@@ -332,3 +332,273 @@ The UDF IDs can be seen in the query plan, for example, ``add1(...)#2L`` in ``Ar
 
 
 This feature is not supported with registered UDFs.
+
+Common Exceptions / Errors
+--------------------------
+
+PySpark SQL
+~~~~~~~~~~~
+
+- **AnalysisException**
```

Comment:
```suggestion
**AnalysisException**
```

Not strong feeling about this, but seems like the indentation is a bit unmatched with the contents below ??

Reply:
Good idea! Indentation is adjusted.